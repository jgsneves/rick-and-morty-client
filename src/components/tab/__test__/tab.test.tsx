import { render } from '@testing-library/react';
import Tab from '..';

describe('Tab test suite', () => {
  it('should render Episodes closed tab', () => {
    const { asFragment } = render(<Tab isOpened={false} name="Episódios" />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should render Episodes opened tab', () => {
    const { asFragment } = render(<Tab isOpened name="Episódios" />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should render Characteres closed tab', () => {
    const { asFragment } = render(<Tab isOpened={false} name="Personagens" />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should render Characteres opened tab', () => {
    const { asFragment } = render(<Tab isOpened name="Personagens" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
