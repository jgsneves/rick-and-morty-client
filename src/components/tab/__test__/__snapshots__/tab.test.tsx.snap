// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab test suite should render Characteres closed tab 1`] = `
<DocumentFragment>
  <button
    class="sc-bczRLJ OuACh"
    type="button"
  >
    Personagens
  </button>
</DocumentFragment>
`;

exports[`Tab test suite should render Characteres opened tab 1`] = `
<DocumentFragment>
  <button
    class="sc-bczRLJ hnxMtX"
    type="button"
  >
    Personagens
  </button>
</DocumentFragment>
`;

exports[`Tab test suite should render Episodes closed tab 1`] = `
<DocumentFragment>
  <button
    class="sc-bczRLJ OuACh"
    type="button"
  >
    Episódios
  </button>
</DocumentFragment>
`;

exports[`Tab test suite should render Episodes opened tab 1`] = `
<DocumentFragment>
  <button
    class="sc-bczRLJ hnxMtX"
    type="button"
  >
    Episódios
  </button>
</DocumentFragment>
`;
